boilerplate/app/models/EpisodeStore.ts

```diff
@@ -1,12 +1,13 @@
-import { Instance, SnapshotOut, types } from "mobx-state-tree"
+import { clone, Instance, SnapshotOut, types } from "mobx-state-tree"
 import { api } from "../services/api"
-import { EpisodeModel } from "./Episode"
+import { Episode, EpisodeModel } from "./Episode"
 import { withSetPropAction } from "./helpers/with-set-prop-action"
 
 export const EpisodeStoreModel = types
   .model("EpisodeStore")
   .props({
     episodes: types.array(EpisodeModel),
+    favorites: types.map(EpisodeModel),
```

Comment:
This isn't bad, but I'm thinking we probably want this to be an array of references instead.

```suggestion
    favorites: types.array(types.reference(EpisodeModel)),
```